src/webview/mod.rs

```diff
@@ -60,6 +60,20 @@ use crate::application::{dpi::PhysicalSize, window::Window};
 
 use http::{Request, Response};
 
+pub struct RequestApi {
+  pub(crate) responder: Box<dyn FnOnce(Response<Cow<'static, [u8]>>)>,
+}
+
+unsafe impl Send for RequestApi {}
+unsafe impl Sync for RequestApi {}
```

Comment:
I don't think these should be implemented in wry.
If it's really required for tokio context. tauri-runtime-wry should be better place.
I believe they will have to write the wrapper again anyway.

Reply:
it's required for the use case this PR covers (spawning another thread to do the operation). If we don't do this, it's useless.

And it's indeed safe to do so (at least it's working).

Reply:
How about just impl Send? And please add documentation to justify why it's safe.